presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -129,9 +133,12 @@ private ConnectorTableMetadata getTableMetadata(String schemaName, String tableN
         result.add(BuiltinColumns.SOURCE.getMetadata());
         result.add(BuiltinColumns.SCORE.getMetadata());
 
+        Map<String, Long> counts = metadata.getSchema().getFields().stream()
+                .collect(Collectors.groupingBy(f -> f.getName().toLowerCase(locale), Collectors.counting()));
```

Comment:
Let's format it as follows:
```suggestion
        Map<String, Long> counts = metadata.getSchema()
                .getFields().stream()
                .collect(Collectors.groupingBy(field -> field.getName().toLowerCase(locale), Collectors.counting()));
```